specificatie/openapi.yaml

```diff
@@ -648,9 +784,89 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    Cirkel:
+      allOf:
+      - $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+      - description: 'Om met een cirkel te zoeken moet je een punt en een straal (in meters) opgeven.'
+      properties:
+        radius:
+          type: number
+          default: 100
+          minimum: 1
+          maximum: 280
+
+#      type: object
+#      properties:
+#        punt:
+#          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+#        radius:
+#          type: number
+#          default: 100
+#          minimum: 1
+#          maximum: 280
+
+    BoundingBox:
+      type: object
```

Comment:
In de [OGC API specs](https://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_parameter_bbox) wordt de bounding box query string parameter als volgt gespecificeerd:

``` yaml
name: bbox
in: query
required: false
schema:
  type: array
  minItems: 4
  maxItems: 6
  items:
    type: number
style: form
explode: false
```

Is het niet beter om dit over te nemen? Of is er een reden om daarvan af te wijken?